packages/components/src/components/RadioCard/RadioCard.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactNode } from 'react';
+
+import styled, { css } from 'styled-components';
+
+import { palette, borders } from '@trezor/theme';
+
+import { Card } from '../Card/Card';
+import { Icon } from '../Icon/Icon';
+import { Box } from '../Box/Box';
+
+export type RadioCardProps = {
+    isActive: boolean;
+    children: ReactNode;
+    onClick: () => void;
+};
+
+const BorderActive = styled.div<{ isActive: boolean }>`
+    position: absolute;
+    inset: 0;
+    box-sizing: border-box;
+    ${({ isActive }) =>
+        isActive &&
+        css`
+            outline: 2px solid ${({ theme }) => theme.borderSecondary};
+            outline-offset: -2px;
+        `}
+    border-radius: ${borders.radii.md};
+    z-index: 1;
+`;
+
+const IconWrapper = styled.div`
+    position: absolute;
+    top: -1px;
+    right: -1px;
+    z-index: 2;
+    border-radius: ${borders.radii.full};
+    background-color: ${({ theme }) => theme.borderSecondary};
+    height: 14px;
```

Comment:
I'd rather not set dimensions like this. Just set the right icon size and only use padding here. Then you won't need the flex properties below.